database/Tenis_stolowy/Tenis_stolowy_create_table.sql

```diff
@@ -1,46 +1,46 @@
 CREATE TABLE [dbo].[Deski](
+	[DeskiID] [int] IDENTITY(1,1) NOT NULL,
 	[Producent] [nvarchar](20) NULL,
 	[Model] [nvarchar](60) NULL,
 	[Waga] [tinyint] NULL,
 	[Cena] [float] NULL,
-	[DeskiID] [int] IDENTITY(1,1) NOT NULL,
 	[Opinie] [varchar](50) NULL,
- CONSTRAINT [pk_Deski_DeskiID] PRIMARY KEY CLUSTERED 
-
+ CONSTRAINT [pk_Deski_DeskiID] PRIMARY KEY (DeskiID)
+ )
 GO
 
 
-CREATE TABLE [dbo].[Okładziny](
+CREATE TABLE [dbo].[Okladziny](
+	[OkladzinyID] [int] IDENTITY(1,1) NOT NULL,
 	[Producent] [nvarchar](max) NULL,
 	[Model] [nvarchar](max) NULL,
-	[Grubość] [float] NULL,
+	[Grubosc] [float] NULL,
 	[Waga] [decimal](18, 0) NULL,
 	[Cena] [float] NULL,
 	[Kolor] [nvarchar](max) NULL,
-	[OkładzinyID] [int] IDENTITY(1,1) NOT NULL,
- CONSTRAINT [pk_Okładziny_OkładzinyID] PRIMARY KEY CLUSTERED 
-
+ CONSTRAINT [pk_Okladziny_OkladzinyID] PRIMARY KEY (OkladzinyID) 
+)
 GO
 
 
-CREATE TABLE [dbo].[Piłeczki](
+CREATE TABLE [dbo].[Pileczki](
 	[Rodzaj] [varchar](20) NULL,
```

Comment:
Tyyy, ale brakuje tutaj klucza głównego w tej tabelce. Poprawiamy to?

Reply:
Dobra, zostawiamy tak jak jest.